tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -162,20 +176,11 @@ def pre_build_jobspecs(self):
     return []
 
   def build_jobspec(self):
-    if self.platform == 'windows':
```

Comment:
nit, optional as fine submitting without: 
can we keep the exception on `windows` here? My thinking is ruby windows artifact still shouldn't be attempted, since the windows builds are just happening under the linux target